packages/react/src/components/ComposedStructuredList/_composed-structured-list.scss

```diff
@@ -0,0 +1,32 @@
+@import '~carbon-components/scss/globals/scss/vars';
+
+.#{$iot-prefix}--composed-structured-list {
+  &__wrapper {
+    width: inherit;
+  }
+
+  &__empty-content {
+    background-color: $ui-01;
+    text-align: center;
+    color: $text-03;
+    // stylelint-disable-next-line declaration-property-unit-blacklist
+    font-size: 14px;
+    padding-top: 90px;
+    padding-bottom: 115px;
+    font-weight: regular;
+    caption-side: bottom;
+    display: table-caption;
+  }
+
+  &__loading {
+    padding-top: $spacing-05;
+  }
+
+  &__list-cell {
+    min-width: var(--width);
```

Comment:
these should probably be more descriptive in the future. While I do not think it should have any conflicts since we are defining the value on the component it may be more self documenting if it were to have `list-cell-width` or some such

Reply:
Yep definitely makes sense. This component will be deprecated in the next release as well